uptime/monitor-setup/uptime-rc-webhook.js

```diff
@@ -46,16 +53,16 @@ class Script {
 
     return {
       content:{
-        alias: USERNAME,
```

Comment:
Looks like you are switching it from spaces to tabs, it's generally better to use space so the indentation don't change in different places. Please revert back!